packages/vue-components/src/directives/Closeable.js

```diff
@@ -2,46 +2,73 @@ module.exports = {
   bind(el) {
     function onClose() {
       el.dataset.isShown = 'false';
-      $showLabel.show();
-      $closeButton.hide();
-      $content.get()[0].style.display = 'none';
+      showLabel.style.display = '';
+      closeButton.style.display = 'none';
+      content.style.display = 'none';
     }
 
     function onShow() {
       el.dataset.isShown = 'true';
-      $showLabel.hide();
-      $content.get()[0].style.display = '';
+      showLabel.style.display = 'none';
+      content.style.display = '';
     }
 
     function onMouseOver() {
       if (el.dataset.isShown === 'false') {
         return;
       }
-      $closeButton.show();
+      closeButton.style.display = '';
     }
 
     function onMouseOut() {
       if (el.dataset.isShown === 'false') {
         return;
       }
-      $closeButton.hide();
+      closeButton.style.display = 'none';
+    }
+
+    function createCloseButton() {
+      const closeButton = document.createElement('span');
+      closeButton.classList.add('closeable-button', 'label', 'label-default', 'hidden-print');
+      closeButton.style.cssText += 'display: none; position: absolute; top: 0; '
+          + 'left: 0; cursor: pointer;background: #d9534f';
+      closeButton.innerHTML = '<span class="glyphicon glyphicon-remove" aria-hidden="true"></span>';
+      closeButton.addEventListener('click', onClose);
+      return closeButton;
+    }
+
+    function createShowLabel(message) {
+      const showLabel = document.createElement('a');
+      showLabel.classList.add('closeable-show', 'hidden-print');
+      showLabel.style.cssText += 'display: none; cursor: pointer;text-decoration: underline';
+      showLabel.innerHTML = message;
+      showLabel.addEventListener('click', onShow);
+      return showLabel;
     }
 
     el.dataset.isShown = 'true';
     el.style.position = 'relative';
+
+    const content = document.createElement('div');
+    content.classList.add('content');
+    Array.from(el.children).forEach(child => content.append(child));
+    el.replaceChildren();
+    el.append(content);
+
+    if (el.className === '') {
+      el.classList.add('closeable-wrapper');
+    } else {
+      el.classList.add(`${el.className} closeable-wrapper`);
```

Comment:
is there a problem here? so if the el.className is not empty, we will add into the el.classList `el.className` and the `closable-wrapper`?

Reply:
Ah yes good point corrected in [745ff4c](https://github.com/MarkBind/markbind/pull/2130/commits/745ff4cbf4ae4d464b39bfac1e6868b24d110f01)!